.gitignore

```diff
@@ -19,3 +19,6 @@ prometheus_rsa
 # Ruby artifacts
 .bundle/
 vendor/
+
+# IDEA
```

Comment:
We try to not have any editor specific things in `.gitignore`. We recommend adding these to your global git `.gitignore` file. More details are e.g. here: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c